packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -19,6 +27,10 @@ export default function ({ types: t }) {
         file.set("helperGenerator", function (name) {
           if (HELPER_BLACKLIST.indexOf(name) < 0) {
             return file.addImport(`${moduleName}/helpers/${name}`, "default", name);
+          } else {
+            const node = getHelper(name);
```

Comment:
This is an attempt to avoid the `import` statements in the final output, like `export * from`, correct?